ldaptor/entry.py

```diff
@@ -116,7 +116,7 @@ def __str__(self):
                 vs = list(values)
                 vs.sort()
                 a.append((key, vs))
-        return ldif.asLDIF(self.dn, a)
+        return ldif.asLDIF(self.dn.toWire().decode('utf-8'), a)
```

Comment:
hm... is this a change in the API?

previously this was distinguishedname.DistinguishedName and now is Unicde... or this is what it looks

Reply:
With the changes in ``ldif`` module ``self.dn`` is passed to ``asLDIF`` function as in the previous versions like in ``delta`` module above.